core/store/src/columns.rs

```diff
@@ -63,6 +63,11 @@ pub enum DBCol {
     /// - *Rows*: peer_id (PublicKey)
     /// - *Content type*: [network_primitives::types::KnownPeerState]
     Peers,
+    /// List of recent long-standing outbound TIER2 connections. We'll attempt to
+    /// reestablish these connections after node restart or upon disconnection.
+    /// - *Rows*: single row (empty row name)
+    /// - *Content type*: Vec of PeerId
+    RecentConnections,
```

Comment:
idea: how about we store it as a protobuf instead? This will simplify extending the storage schema in the future.